.github/actions/setup-env/action.yml

```diff
@@ -0,0 +1,53 @@
+name: Setup Package Environment
+description: Based on the provided package name, this action will map and set up the necessary environment variables
+inputs:
+  pkg_name:
+    description: 'The package this will setup the environment for'
+    required: true
+
+runs:
+  using: composite
+  steps:
+      - name: Setup env variables based on pkg_name
+        env:
+          PACKAGE_NAME: ${{ inputs.pkg_name }}
+        run: |
+          local WORKSPACE_PATH=''
+          local PROJECT_FILE=''
+          local BUILD_OUTPUT_PATH=''
+          local BUILD_OUTPUT_DLL_NAME=''
+          local TEST_PROJECT_FILE=''
+          local CONTRACT_TEST_PROJECT_FILE=''
+          local CONTRACT_TEST_DLL_FILE=''
+          
+          if [[ $PACKAGE_NAME == 'LaunchDarkly.ClientSdk' ]]; then
+            WORKSPACE_PATH='pkgs/sdk/client'
```

Comment:
If feels like this configuration should live with the specific project. like a `package.env` file or something. So you can remove the hotspot.

Reply:
Done